lib/health-data-standards/tasks/bundle.rake

```diff
@@ -227,15 +227,26 @@ namespace :bundle do
   end
 
   desc 'Remove all artifacts from NLM import process'
-  task :reset => :environment do
-    [Record,
-     HealthDataStandards::CQM::Bundle,
-     HealthDataStandards::CQM::Measure,
-     HealthDataStandards::CQM::PatientCache,
-     HealthDataStandards::CQM::QueryCache,
-     HealthDataStandards::SVS::ValueSet].each do |klass|
-       num_deleted = klass.delete_all
-       puts "#{klass.name}: Deleted [#{num_deleted}] records"
-     end
+  task uninstall: :environment do
```

Comment:
`bundle:uninstall` calls `bundle:cleanup` explicitly in line 231. Is that what you mean?